lib/src/main/java/com/orange/ods/compose/component/bottomnavigation/OdsBottomNavigation.kt

```diff
@@ -23,15 +31,22 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.ImageBitmap
 import androidx.compose.ui.graphics.painter.Painter
 import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.semantics.Role
 import androidx.compose.ui.text.style.TextOverflow
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import com.orange.ods.R
 import com.orange.ods.compose.component.OdsComposable
 import com.orange.ods.compose.component.content.OdsComponentIcon
 import com.orange.ods.compose.component.content.OdsComponentScopeContent
 import com.orange.ods.compose.component.utilities.Preview
 import com.orange.ods.compose.component.utilities.UiModePreviews
 import com.orange.ods.compose.theme.OdsTheme
 
+private const val BottomNavigationItemMaxCount = 5
```

Comment:
```suggestion
private const val MaxBottomNavigationItemCount = 5
```

We usually put `Max` at the beginning of the name with count. For instance: `MaxActionButtonCount` and `MaxNavigationItemCount`.